osu-events-report.py

```diff
@@ -79,18 +79,37 @@ def create_report_by(fields):
 
 
 def main():
-    global output, events, end, start
+    global output, events, end, file_name, start
```

Comment:
Since we are now calling `get_events()` multiple times, I think `start` and `end` shouldn't be global variables.